win32/test/test_odbc.py

```diff
@@ -206,7 +206,7 @@ def testLongBinary(self):
 
     def testRaw(self):
         ## Test binary data
-        self._test_val("rawfield", memoryview(b"\1\2\3\4\0\5\6\7\8"))
+        self._test_val("rawfield", memoryview(b"\1\2\3\4\0\5\6\7\\8"))
```

Comment:
why just one of the slashes here?

Reply:
`\8`` is not a valid escape sequence, `\0` to `\7` is. This keeps the same behaviour:
```py
>>> b"\0\1\2\3\4\5\6\7\8"
b'\x00\x01\x02\x03\x04\x05\x06\x07\\8'
```